frontend/src/theme/mainTheme/mainColors.ts

```diff
@@ -0,0 +1,40 @@
+import { transparentize } from "color2k"
+import { colors } from "../primitives/colors"
+
+const genericColors = {
+  ...colors,
+  bodyText: colors.gray90,
+  danger: "#9d292d",
+  info: "#1e6777",
+  success: "#176c36",
+  warning: "#947c2d",
+  primary: "#f63366",
```

Comment:
I don't love this but this is probably/hopefully something I'll get to tackle :)